packages/now-node/test/fixtures/48-change-handler-by-env/now.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": 2,
+  "builds": [
+    {
+      "src": "index.js",
+      "use": "@now/node"
+    }
+  ],
+  "build": {
```

Comment:
You are using a build environment variable but this must be a runtime environment variable since helpers are injected at runtime.